spec/requests/profiles_controller_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+describe ProfilesController do
+  let(:user)        { create(:user) }
+  let!(:other_user) { create(:user) }
+  before { login_as(user) }
+
+  describe '#show_self' do
+    it 'shows the user\'s own profile' do
+      get "/profiles/#{user.screen_name}"
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    context 'when the screenname is invalid' do
+      it 'redirects to own profile' do
+        get '/profiles/dummy'
+
+        expect(response).to redirect_to(profiles_me_path)
+      end
+    end
+  end
+
+  describe '#show' do
+    it 'shows the own profile of the user' do
```

Comment:
I think you wanted to put this to the description of the `#show_self` and to here without the ‘own’.